web/src/client/storage.js

```diff
@@ -117,25 +118,33 @@ class DevicesManager {
    * @property {string} [transport]
    * @property {boolean} [sdCard]
    * @property {boolean} [dellBOOS]
-   * @property {string[]} [devices] - RAID devices (only for "raid" type)
+   * @property {string[]} [devices] - RAID devices (only for "raid" and "md" types)
    * @property {string[]} [wires] - Multipath wires (only for "multipath" type)
    * @property {string} [level] - MD RAID level (only for "md" type)
    * @property {string} [uuid]
-   * @property {string[]} [members] - Member devices for a MD RAID (only for "md" type)
    * @property {boolean} [active]
    * @property {string} [name] - Block device name
    * @property {number} [size]
+   * @property {number} [recoverableSize]
    * @property {string[]} [systems] - Name of the installed systems
    * @property {string[]} [udevIds]
    * @property {string[]} [udevPaths]
-   * @property {PartitionTableData} [partitionTable]
+   * @property {PartitionTable} [partitionTable]
+   * @property {Filesystem} [filesystem]
    *
-   * @typedef {object} PartitionTableData
+   * @typedef {object} PartitionTable
    * @property {string} type
+   * @property {StorageDevice[]} partitions
+   * @property {number} unpartitionedSize - Total size not assigned to any partition
+   *
+   * @typedef {object} Filesystem
+   * @property {string} type
+   * @property {boolean} isEFI
```

Comment:
after this change I have to read https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html to learn how it works and what means that square brackets :+1: